stable/redis-ha/templates/redis-ha-configmap.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-configmap
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+data:
+  redis.conf: |
+    dir "/data"
+    {{- range $key, $value := .Values.redis.config }}
+    {{ $key }} {{ $value }}
+    {{- end }}
+{{- if .Values.auth }}
+    requirepass replace-default-auth
+    masterauth replace-default-auth
+{{- end }}
+
+  sentinel.conf: |
+    dir "/data"
+    {{- $root := . -}}
+    {{- range $key, $value := .Values.sentinel.config }}
+    sentinel {{ $key }} {{ $root.Values.redis.masterGroupName }} {{ $value }}
```

Comment:
It can only express sentinel config like `sentinel xxx mymaster xxx`.
However, sentinel have some other config not to  fulfil this rule.

e.g.
`logfile ""`
https://github.com/antirez/redis/blob/unstable/sentinel.conf#L36

Reply:
@ceshihao yeah you are correct. I assumed things like `logfile` or `pidfile` would be unnecessary to modify in a Kubernetes context. I can make a note of this that not all config values are supported.

The other option would be to allow to pull the entire config from the values.yaml file. (Which might be a good option anyway).

Any suggestions?

Reply:
I think move the entire config to `values.yaml` is a better option and give the flexibility to users.